src/components/Table/Table/__tests__/table.jest.tsx

```diff
@@ -69,6 +71,18 @@ describe("Table", () => {
     });
   });
 
+  describe("TableHeader", () => {
+    it("should have default medium height", () => {
+      const { getByRole } = render(<TableHeader />);
+      expect(getByRole("rowgroup")).toHaveStyle(`height: ${RowSizes.MEDIUM}px`);
+    });
+
+    it("should have large height", () => {
+      const { getByRole } = render(<TableHeader size={RowSizes.LARGE} />);
+      expect(getByRole("rowgroup")).toHaveStyle(`height: ${RowSizes.LARGE}px`);
+    });
+  });
```

Comment:
we have visual tests through chromatic that go over each story and capture it, so you can remove these tests

Reply:
done